tests/features/paragraphs/groupex_pro_paragraphs.feature

```diff
@@ -31,6 +31,54 @@ Feature: GroupEx Pro paragraphs
     And Element "#edit-location--wrapper .fieldset-legend" has text "Locations"
     And I should see an "#edit-location" element
     And I should see an "#edit-location .form-item-location" element
+    When I select "202" from "location"
+    Then I wait for AJAX to finish
+    And I select this "next monday" from "date_select"
+    Then I wait for AJAX to finish
+    And I should see an ".groupex-pdf-link-container a:contains('Download PDF')" element
+    And I should see text matching "8:30am 60 min"
+    And I should see text matching "Chisel Studio 1"
+    And I should see text matching "Jennifer K."
+    And I should see text matching "Chisel Visit Strength with Jennifer K. Class will take place at Studio 1. GXP Club - Fable Jennifer K."
+    And I should see text matching "11:00am 60 min"
+    And I should see text matching "Yoga GX2"
+    And I should see text matching "John F."
+    And I should see text matching "Yoga Visit Yoga with John F. Class will take place at GX2. GXP Club - Fable John F."
+    And I should see text matching "7:00pm 60 min"
+    And I should see text matching "3D XTREME­&trade; GX1"
```

Comment:
This " &trade" should be rendered as ™  . 
So I guess we should fix the error. And update test on all those places.